client/my-sites/upgrades/domain-management/email/index.jsx

```diff
@@ -41,8 +43,14 @@ const Email = React.createClass( {
 	},
 
 	render() {
+		const showPlanFeatures = config.isEnabled( 'manage/plan-features' );
+
+		const classes = classNames( 'domain-management-email', {
+			'is-wide-layout': showPlanFeatures
```

Comment:
I did it the same way as in `my-sites/plans/main.jsx`. Should I refactor that, too? Also, in that file, `has-personal-plan` is appended to the classes list if personal plan is enabled. Should that be prop of `Main`, too?